### PR TITLE
update release notes

### PR DESCRIPTION
**Pull request by jukent**

### CODE REVIEW

**Comment by jukent** [comment]
Full changelog link might still break until release is made? How is this handled?

**Review by pilotchute — APPROVED** [review]

File: docs/release-notes.rst [hunk 1]
```diff
@@ -4,11 +4,25 @@
...
+^^^^^^^^^^^^
+* Pressure-height twin axis function `add_height_from_pressure_axis` in (:pr:`114`)
 
 Documentation
 ^^^^^^^^^^^^^
+* Contributor's Guide updated in (:pr:`115`)
+* Documentation and Packaging Overhaul to match GeoCat-Comp style in (:pr:`116`), (:pr:`121`)
+* Adding examples links to all functions in (:pr:`122`)
+
+**Full Changelog**: https://github.com/NCAR/geocat-viz/compare/v2023.03.0...v2023.05.0
```
Comment (jukent) [inline docs/release-notes.rst:22]: @pilotchute Is this the right thing to do for the changelog link? That link doesn't exist until the release.

Comment (anissa111) [reply docs/release-notes.rst:22]: @jukent Yes, that's correct. We're actually omitting this in geocat-comp now, but this was the way we did it up until this coming release.

Comment (jukent) [reply docs/release-notes.rst]: Okay I'll follow suit and remove it here too. The broken link seems annoying every time.

**Review by kafitzgerald — APPROVED** [review]

**Pull request merged**